src/main/java/stream/flarebot/flarebot/commands/random/JumboCommand.java

```diff
@@ -56,4 +51,18 @@ public String getUsage() {
     public CommandType getType() {
         return CommandType.RANDOM;
     }
+    
+    private InputStream read(String url, TextChannel channel) {
```

Comment:
If we can we might want to put a max size on this

Reply:
What do you mean?

Reply:
I Imagin file size and image size is bigger for emotes then it is for discord files so I think we'd run into errors if someone has a big emote

Reply:
Max emote size is 256KB

Reply:
nvm then